gitspy/Parseable.swift

```diff
@@ -9,7 +9,31 @@
 import Foundation
 
 protocol Parseable {
-    typealias A
-    func parse(data: AnyObject) -> A?
+    
+    init?(dictionary: [String: AnyObject])
+    init?(data: AnyObject)
```

Comment:
@joanromano not sure, maybe no but ATM it's just a default implementation that convert the object to dictionary... we can just remove it.
I think for parsing the bailable initializer is the right way to go, agree?